app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt

```diff
@@ -39,6 +39,8 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
     private var mSearchAction: MenuItem? = null
     private var isSearchOpened = false
     private var edtSearch: EditText? = null
+    var text: String = ""
+    var afterText: String = ""
```

Comment:
This variable can be local, I still have no idea why is this global?

Please tell me the amount of places this variable is used and if there is only one coding block where this is being used, then please make this local as I've requested

Also, please tell me where is this modified, if it is not modified since the declaration, make this immutable